### PR TITLE
Rewrite of icon section of the HIG

### PR DESCRIPTION
**Pull request by snwh**
It's a rewrite of the icon section of the HIG.

### CODE REVIEW

File: docs/human-interface-guidelines.md [hunk 1]
```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
...
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/utilities-terminal.svg">
+</div>
 
-### Design Elements {#design-elements}
 
-#### Shape {#shape}
+### Metaphors
 
-Icons should have a distinctive shape/silhouette to improve recognition. The shape should not be too complicated, but not every icon should be a rounded rectangle.
+If you're creating a icon for a hardware device or a file type (such as those for MimeType or Device icons), its shape is typically a visual representation of its real-world counterparts. For example, the icon for a camera is a stylized camera.
```
Comment (lewisgoddard) [inline docs/human-interface-guidelines.md]: I believe this should be:
> If you're creating **an** icon for a hardware device or a file type ...

File: docs/human-interface-guidelines.md [hunk 2]
```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
...
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/video-display.svg">
+</div>
 
-All elementary Icons, and shapes within the icons, are stroked with a thin outline to improve contrast. At low resolutions the stroke size is 1px. The stroke should _not_ scale along with the icon, it should remain 1px.
 
-The color of the outline is a dark variant of the key color of the icon. This is done by creating a stroke at 30% opacity of black such that the border of the background lines up at half pixels. This is what allows icons with a stroke to appear sharp on dark backgrounds as well as light ones.
+### Action Icons
 
-#### Highlights {#highlights}
+Action icons are used to represent common user actions, such as "delete", "play" or "save". These icons are mostly found in app toolbars, but can be found throughout the OS.
```
Comment (lewisgoddard) [inline docs/human-interface-guidelines.md]: Do we use oxford commas? We really should.

> ... such as "delete", "play"**,** or "save".

"mostly found" might be better phrased as "most often found", and you don't need to mention they are found elsewhere if you've already said "mostly", it implies there are others for you.

**Comment by danrabbit** [comment]
- [ ] On mobile, your two big grid images of the videos, terminal, installer, etc icons align right instead of center

File: docs/human-interface-guidelines.md [hunk 3]
```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
...
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/edit-cut.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/edit-undo.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/object-inverse.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/media-playback-start.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/tag-new.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/open-menu.svg">
+</div>
 
-#### Limited glossy reflections {#limited-glossy-reflections}
+If your app makes use of a one of these common actions, reference its corresponding icon instead of creating your own. This ensures a consistent user experience and aids in user recognition of common functions.
```
Comment (lewisgoddard) [inline docs/human-interface-guidelines.md]: > ... of a one ...
> ... reference it's corresponding ...

(possessive)

File: docs/human-interface-guidelines.md [hunk 4]
```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
...
 
-Use glossy reflection only on objects that have a reflective surface in real life (plastic, glass, some metal, et cetera). A sheet of paper should not be glossy.
+However if your app has a unique action, you may need to create your own. When doing this, try to follow the look and feel of existing action icons and include it along with your app.
```
Comment (lewisgoddard) [inline docs/human-interface-guidelines.md]: _However_ should probably have a comma after it.

File: docs/human-interface-guidelines.md [hunk 5]
```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
...
 
-### Lighting and Perspective {#lighting-and-perspective}
 
-elementary Icons have a consistent light source and perspective to keep a similar look throughout the system. The icons have been designed with the UI theme in mind, meaning everything shares a cohesive feel and a subtle amount of realism.
+## Icon Sizes {#size}
 
-#### Lighting {#lighting}
+elementary OS uses **six** main icon sizes throughout the desktop. So if you're developing an application for elementary or making an icon set, it's best to include all six.
```
Comment (lewisgoddard) [inline docs/human-interface-guidelines.md]: > So if you're developing an application for elementary...

That sentence puts me on edge. _"So if you're..."_ sounds too friendly, and this is the HIG, who would be here except developers?

File: docs/human-interface-guidelines.md [hunk 6]
```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
...
+| 32x32         | 2             | 26       | 16               |
+| 48x48         | 3             | 40       | 24               |
+| 64x64         | 4             | 56       | 32               |
+| 128x128       | 9             | 104      | 64               |
 
-App launcher icons are the first look that prospective users will get of your app in the software center. A high quality app icon can influence users to find out more as they scroll through lists of apps.
+### Exceptions
 
-Quality matters here; a well-designed icon can be a strong signal that your app is of similarly high quality. Consider working with an icon designer to develop the app’s launcher icon.
+However there are exceptions. Many icons (especially mimetype icons) have ascending and descending elements -those that extend beyond the base line and x-height line (shown here in <span style="color:orange;">orange</span>.)
```
Comment (lewisgoddard) [inline docs/human-interface-guidelines.md]: space dash space ?

Comment (danrabbit) [inline docs/human-interface-guidelines.md]: Probably should just be a colon since it's not interjecting right? Otherwise I guess it would be an em dash

**Pull request merged**